packages/audit/scripts/analyze-cli-usage.py

```
@@ -0,0 +1,223 @@
+        raise ValueError(msg)
+
+    calls = [e for e in events if e.get("type") == "cli_call"]
+    if days is None or days == 0:
+        return calls
+    cutoff = datetime.datetime.now(datetime.UTC) - datetime.timedelta(days=days)
+    cutoff_str = cutoff.isoformat()
+    return [c for c in calls if c.get("ts", "") >= cutoff_str]
+
+
+def extract_keywords(prompt: str) -> list[str]:
+    """プロンプトから主要キーワードを抽出する（簡易版）。
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)